certified-connectors/Leadinfo/apiDefinition.swagger.json

```diff
@@ -0,0 +1,293 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Leadinfo",
+    "description": "Leadinfo enables you to see real-time which companies are visiting your website and what pages they're viewing. By installing this app you can easily share recognized companies with your team from our portal! You need a Leadinfo account to use this App. You can request a free 14-day trial on our [website](https://portal.leadinfo.com/registration/?utm_source=microsoftteams).\n\nContact us [here](https://www.leadinfo.com/en/contact/) if you need any help.",
```

Comment:
@GerbenZ01 is it possible to remove the special characters "\n" from the description?
I would suggest you have a proper description for the connector. and for this contact information you can have them in the contact section or in your Intro.md file. 
We will ask you to submit the Intro.md file once this PR is approved and it is time to submit to ISV Studio. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6